mod.conf

```diff
@@ -3,4 +3,4 @@ description = Scratch programmable robots.
 title = LWScratch
 name = lwscratch
 depends =
```

Comment:
not related to the pr, but mind getting rid of this line? doesnt need to be here since its blank